docs/proposals/device-crd.md

```diff
@@ -280,25 +319,20 @@ type DeviceStatus struct {
 // the actual state to the cloud. Offline device interaction in the edge is possible via twin
 // properties for control/command operations.
 type Twin struct {
-  // The property name for which the expected/actual values are reported.
+  // Required: The property name for which the expected/actual values are reported.
   // This property should be present in the device model.
-  // Required.
   Name     string    `json:"name,omitempty"`
-  // the expected attribute value for this property.
-  // Required.
+  // Required: the expected attribute value for this property.
   Expected Attribute `json:"expected,omitempty"`
```

Comment:
should we rename Attribute to TwinProperty ?

Reply:
Yes we can